packages/block-editor/src/store/selectors.js

```diff
@@ -1382,10 +1382,28 @@ const canInsertBlockTypeUnmemoized = (
 		parentName
 	);
 
+	let hasBlockAllowedAncestor = true;
```

Comment:
Hi!

Do we need a default value here?

Reply:
Yes because if not `canInsert` will always be false if `blockType.ancestor` is falsy.